enc_temp_folder/ec1feb469841afeb2e47f7b975f/ColouringBookGameMode.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 1998-2017 Epic Games, Inc. All Rights Reserved.
```

Comment:
Yeah, I committed the file before adding that folder to .gitignore. 

Deleting...